graphql_compiler/tests/test_emit_output.py

```diff
@@ -243,8 +248,8 @@ def test_output_inside_optional_traversal(self):
                     BinaryComposition(
                         u'!=',
                         # HACK(predrag): The type given to OutputContextVertex here is wrong,
-                        # but it shouldn't cause any trouble since it has absolutely nothing to do
-                        # with the code being tested.
+                        # but it shouldn't cause any trouble since it has absolutely nothing to
```

Comment:
Any particular reason for this change?

Reply:
Linter was complaining about line lengths

Reply:
The old line length was 99, and the linter I believe is set to enforce line length under 100. Is this your local linter? If so, can you double-check your linter configuration?

Reply:
Hmm, that's strange-- when I ran the linter originally, it did complain, but after changing it back, it stopped complaining. 

My best guess as to what happened is that an extra space or two snuck in by accident which is why it got upset -> I change the line length -> then get rid of the extra whitespace. In any case, that's resolved!